### PR TITLE
Piole007

### PR DESCRIPTION
**Pull request by piole007**
Hey! I added a nav-bar with links, a photo gallery for each season and About Us page.  There's one problem - the photo gallery appears on one part of main section's text. I ma so sorry for breaking your code :( 

Anyway, I will try to fix this and I'm not going to merge this until the solution is found. Just thought you might wanna take a look how the site looks right now.

Also, I didn't add 'Home' button, trying to keep the page as clean as possible. Instead I added a link to Logo, so when you click on it, you are taken to the main view. Let me know what your thoughts are on this.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,43 +20,63 @@
...
-</div>
     <!-- PHOTO SECTION -->
-
+    <div class="photo-wrapper"></div>
+    <div id="spring"></div>
+  <div id="summer"></div>
+<div id="fall"></div>
+<div id="winter"></div>
+</div>
     <!-- FOOTER -->
```
Comment (IevaZR) [inline index.html:80]: I can't see the pictures :(

**Review by IevaZR — APPROVED** [review]
Looks nice :)

File: stylesheet.css [hunk 2]
```diff
@@ -1,97 +1,134 @@
...
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 6rem;
+  width: 100%;
+  background-color: #ddbea9;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 2rem;
```
[the remark this answers is not in the document]
Comment (piole007) [reply stylesheet.css:16]: Great idea!

File: index.html (hunk 1 above)
Comment (piole007) [reply index.html:80]: Yes, the files where not pushed up properly. Should be fixed with newest commit.

**Pull request merged**